src/skin/tooltips.cpp

```diff
@@ -723,18 +751,34 @@ void Tooltips::addStandardTooltips() {
             << tr("Next or Previous")
             << tr("Switch to either the next or previous effect.");
 
+    add("EffectSlot_metaknob")
+            << tr("Meta Knob")
+            << tr("Controls linked parameters of this effect")
+            << QString("%1: %2").arg(rightClick, resetToDefault);
+
+    add("EffectSlot_focus")
+            << tr("Effect Focus Button")
+            << QString("%1: %2").arg(leftClick, tr("Focuses this effect."))
+            << QString("%1: %2").arg(rightClick, tr("Unfocuses this effect."))
+            << tr("Refer to the web page on the Mixxx wiki for your controller for more information.");
+
     add("EffectSlot_parameter")
             << tr("Effect Parameter")
             << tr("Adjusts a parameter of the effect.")
             << QString("%1: %2").arg(rightClick, resetToDefault);
 
     add("EffectSlot_parameter_link_type")
```

Comment:
That tooltip refers to a specific button style in a skin. Then it should be a requirement to have skins in master the button styled that way, see https://bugs.launchpad.net/mixxx/+bug/1531519

We have something similar in  https://github.com/mixxxdj/mixxx/blob/master/src/skin/tooltips.cpp#L534 , and over the time skins developed to not work all as described.